_posts/2019-03-22-aerogear-mobile-services-2.0.0-release.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: news
+section: news
+title: AeroGear Mobile Services version 2.0.0 is here
+author: aerogear
+tag: community
+---
+
+About 7 months ago, we announced [the 1.0.0 release](/news/2018/07/04/aerogear-mobile-services-announcement/index.html) of the AeroGear Mobile Services. Since then we have been working very hard to continue the development work. Today we are delighted to announce the availability of AeroGear Mobile Services version 2.0.0.
+
+## What's New
+
+* A new [Data Sync](/services/data-sync/) framework is now available. It is built on top of the popular [Apollo GraphQL framework](https://www.apollographql.com/). We added a set of new features like offline support, conflict handling etc to make it easier for developers to create mobile applications that need to sync data between clients and server. To learn more, please checkout the [Voyager server](https://github.com/aerogear/voyager-server) and the [Voyager client](https://github.com/aerogear/aerogear-js-sdk/tree/master/packages/sync) repos.
+* A new [Cordova showcase app](https://github.com/aerogear/ionic-showcase) is created to demonstrate the new data sync capabilities and how to use it with other services.
```

Comment:
```suggestion
* A new [Cordova showcase app](https://github.com/aerogear/ionic-showcase) is created to demonstrate the new data sync capabilities and how to use it with other services.
```
* A new [Cordova showcase app](https://github.com/aerogear/ionic-showcase) was created to demonstrate the new data sync capabilities and how to use it with other services.